modules/network-flow-matrix.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/configuring-firewall.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="network-flow-matrix_{context}"]
+= {product-title} network flow matrix
+
+The network flow matrix describes the ingress flows to {product-title} services.
+The network information in the matrix is accurate for both bare-metal and cloud environments.
+Use the information in the network flow matrix to help you manage ingress traffic.
+You can restrict ingress traffic to essential flows only to improve network security.
+
+To view or download the raw CSV content, see link:https://raw.githubusercontent.com/rohennes/openshift-docs/TELCODOCS-1670/snippets/network-flow-matrix.csv[this resource].
```

Comment:
I don't know if Telco is subject to the requirement about [not linking to github repos](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites) - I know some teams allow it, but I thought I'd ask.

Reply:
Thanks Kerry. This is autogenerated content that we would like to embed directly using the [rawgithub usercontent method](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#embedding-an-external-file) in 4.17. For now, the automation of this is not fully complete - we need to edit a few fields by hand - so this is a halfway house that will be updated in 4.17. Hope that works.